notes/1.0.3/watch.md

```diff
@@ -0,0 +1,10 @@
+### Fixes with compatibility implications
+
+### Improvements
+
+### Bug fixes
+
+- Add `sbt.watch.polling` system property to allow switching back to old polling behaviour for watch. This is a workaround for [#3527][3527] by [@strinbean][@stringbean].
```

Comment:
Could you describe what happens if someone passed in `true`, otherwise, and what the default value is.